examples/kustomization.yaml

```diff
@@ -25,7 +25,7 @@ configMapGenerator:
 resources:
   - certificate-authority.yaml
   - sa.yaml
-  - github.com/utilitywarehouse/cockroachdb-manifests//base?ref=v23.1.5-1
+  - github.com/utilitywarehouse/cockroachdb-manifests//base?ref=v23.1.10
```

Comment:
We should put in the suffix `-1`, so the final would be `v23.1.10-1` as the last one is for the version of the manifests

Reply:
I'll add a `-0` - seems like it's been indexed from `0` in the past.

Reply:
We base ours loosely on https://wiki.archlinux.org/title/Arch_package_guidelines#Package_versioning

But we prefix versions with `v`. But we do follow:

> release count starts at 1

Reply:
Can you point me to an example?

Following the same versioning across `uw` would be nice. 

I'm ok with with starting from `1`, but given the tagging history I'll diverge to `dev-enablement`.

Reply:
https://github.com/utilitywarehouse/grafana-manifests/releases

I don't feel strongly about it, my comment only applicable if the choice was completely open ended.

Reply:
We started doing the same for https://github.com/utilitywarehouse/kafka-manifests/releases and https://github.com/utilitywarehouse/opentelemetry-manifests/releases

Reply:
> We should put in the suffix -1, so the final would be v23.1.10-1 as the last one is for the version of the manifests

The tagging+release process should be documented in this repo (or notes that it uses some conventional releasing process and link to the docs outlining that process)